docs/topics/multiplatform-mobile/multiplatform-mobile-ios-dependencies.md

```diff
@@ -194,7 +194,9 @@ import DateTools.*
 
 5. Provide values for these two mandatory properties:
     * `modules` – the name of the framework that should be processed by the cinterop.
-    * `package` – the name of the package these declarations should be put into. For example:
+    * `package` – the name of the package these declarations should be put into.
+
+   For example:
```

Comment:
JFYI

`For example` should be in next paragraph, just like in section "Add a library without CocoaPods"